sdk/eventhubs/azure-messaging-eventhubs/migration-guide.md

```diff
@@ -1,142 +1,148 @@
 
-In the interest of simplifying the API surface, we've made three clients, each with an asynchronous and synchronous
-variant. One client is for producing events, `EventHubProducerAsyncClient`, while two are intended for reading events.
-`EventProcessorClient` is the production-level consumer and `EventHubConsumerAsyncClient` for exploration and
-lower-level control of `EventData` consumption.
+- Using the new azure-identity library to share a single authentication approach between clients.
+- A unified logging and diagnostics pipeline offering a common view of the activities across each of the client libraries.
+- A unified asynchronous programming model using [Project Reactor][project-reactor].
 
-[EventProcessorClient][EventProcessorClient] supports checkpointing and load balancing using a plugin model.
-Currently, only Azure Storage Blobs is supported through
-[azure-messaging-eventhubs-checkpointstore-blob][azure-messaging-eventhubs-checkpointstore-blob], but support for other
-durable storage (i.e. Cosmos DB, Redis) may be added in the future.
+## Important changes
 
-| Operation | Asynchronous client | Synchronous client |
-|---|---|---|
-| Producing events | [EventHubProducerAsyncClient][EventHubProducerAsyncClient] | [EventHubProducerClient][EventHubProducerClient] |
-| Consuming events (supports checkpointing and load balancing) | [EventProcessorClient][EventProcessorClient] | |
-| Consuming events | [EventHubConsumerAsyncClient][EventHubConsumerAsyncClient] | [EventHubConsumerClient][EventHubConsumerClient] |
+### Package names and namespaces
 
-### Converting core classes
+Group ids, artifact ids, and package names for the modern Azure client libraries for Java have changed. Each will have the group id `com.azure`, an artifact id following the pattern `azure-[area]-[service]`, and the root package name `com.azure.[area].[Service]`. The legacy clients have a group id `com.microsoft.azure` and their package names followed the pattern `com.microsoft.azure.[service]`. This provides a quick and accessible means to help understand, at a glance, whether you are using modern or legacy clients.
+
+In Event Hubs, the modern client libraries have packages and namespaces that begin with `com.azure.messaging.eventhubs` and were released starting with version 5. The legacy client libraries have package names beginning with `com.microsoft.azure.eventhubs` and a version of 3.x.x or below.
+
+### Client hierarchy and constructors
+
+In the interest of simplifying the API surface, we've made three clients, each with an asynchronous and synchronous variant. One client is for producing events, [EventHubProducerAsyncClient][EventHubProducerAsyncClient], while two are intended for reading events. [EventProcessorClient][EventProcessorClient] is the production-level consumer and [EventHubConsumerAsyncClient][EventProcessorClient] for exploration and
```

Comment:
> we've made three clients, each with an asynchronous and synchronous variant

I thought EventProcessorClient  does not have the two variations

> EventProcessorClient is the production-level consumer

This would make the reader feel that the other client is not meant to be used in production. That can be misleading...

How about this:

> EventProcessorClient is what you would typically use to read events from all partitions with an optional, but recommended checkpointing feature. EventHubConsumerAsyncClient is what you would use for lower-level control of EventData consumption from a single partition.